civihr_default_theme/scss/base/_variables.scss

```diff
@@ -34,6 +34,7 @@ $gray                 : #767b8e; // main text/lighter, menu item bg hover, foote
 $gray-dark            : #555862; // menu bg, default: lighten(#000, 20%) !default
 $gray-darker          : #4c4c5d; // text darker, default: lighten(#000, 13.5%) !default; // #222
 $crm_gray-darker-alt  : #404454; // page title
+$crm_gray-border      : #777C8F; // page title
```

Comment:
The name of the variable isn't good, it's referring to a specific element (`border`) while by following bootstrap convention it should convey only the color (`gray`) and if necessary a specific shade of it (`darker`, `light`, etc)

This name is kinda pointless also by the fact that you're later using it for things that are not a border:
```scss
.btn-default {
  border-color: $crm_gray-border;
  color: $crm_gray-border;

  &.active {
    border-color: $crm_gray-border;
    background: $crm_gray-border !important;
    color: $light !important;
  }
}
```
so no sense in calling it that. Please give it an appropriate name